### PR TITLE
Generate changelog for 3.2.1

### PR DESCRIPTION
**Pull request by sergepetrenko**
Also, remove unreleased/ entries and add a new downgrade version to downgrade versions list.

NO_DOC=changelog
NO_TEST=changelog
NO_CHANGELOG=changelog

### CODE REVIEW

**Review by Totktonada — APPROVED** [review]
I see no flaws. Thanks!

File: changelogs/3.2.1.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+* Fixed a memory leak in IPROTO on execution of the callback installed with
+  `box.iproto.override()` (gh-10477).
+* Fixed a memory leak on foreign key constraint check failure (gh-10476).
+* Fixed a memory leak on an attempt to create or modify a user denied due to
+  access restrictions (gh-10485).
+* Fixed a crash when `slab_alloc_factor` is low and memory pressure is high
+  (gh-10148).
+* Fixed a crash in swim on Tarantool shutdown (gh-10495).
+* Fixed a crash when the first of two consequent DDL operations gets
+  rolled back due to WAL failure (gh-10235).
```
Comment (xuniq) [inline changelogs/3.2.1.md:42]:
```suggestion
* Fixed a crash when `slab_alloc_factor` was low and memory pressure was high
  (gh-10148).
* Fixed a crash in swim on Tarantool shutdown (gh-10495).
* Fixed a crash when the first of two consequent DDL operations was
  rolled back due to WAL failure (gh-10235).
```

File: changelogs/3.2.1.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+* Fixed an inconsistency between internal caches and system spaces with
+  MVCC enabled that could lead to unexplainable errors (gh-10262).
+
+
+### Memtx
+
+* Fixed a crash on transaction rollback if memory usage is close to the limit
```
Comment (xuniq) [inline changelogs/3.2.1.md:49]:
```suggestion
* Fixed a crash on transaction rollback if memory usage was close to the limit
```

File: changelogs/3.2.1.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+  could lead to a crash or invalid query results (gh-10558).
+* Added a fiber slice check to `index.count()` to prevent it from blocking
+  for too long while counting tuples in a space stored in memory (gh-10553).
+* Fixed a bug when an attempt to alter the primary index of an empty space
+  triggered a crash if executed concurrently with a DML request (gh-10603).
+
+### Replication
+
+* Fixed a memory leak on disconnection from replica (gh-10480).
+* Fixed a memory leak on a replica when it is removed from the cluster
```
Comment (xuniq) [inline changelogs/3.2.1.md:75]:
```suggestion
* Fixed a memory leak on a replica when it was removed from the cluster
```

File: changelogs/3.2.1.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+#### Xlog
+* Fixed a memory leak in `xlog.pairs()` on failure to open the file (gh-10479).
+
+#### Iconv
+* Fixed a memory leak in `iconv.new()` (gh-10487).
+
+#### Datetime
+
+* Fixed a bug that caused `datetime.parse()` ignore `tzoffset`
+  option if a custom format was used (gh-8333).
```
Comment (xuniq) [inline changelogs/3.2.1.md:112]:
```suggestion
* Fixed a bug that caused `datetime.parse()` ignore the `tzoffset`
  option if a custom format was used (gh-8333).
```

**Review by xuniq — APPROVED** [review]

**Pull request merged**